3-extra/1-card-vailidator.js

```diff
@@ -0,0 +1,36 @@
+//num.toString()            --->all parament value become number
+//isLengthSixteen           ---against--->Number must be 16 digits
+//isAllNumber               ---against--->invalid characters
+//isMornThanTwoTypeNumber   ---against--->only one type of number
+//isSumGreaterEqualSixteen  ---against--->sum less than 16
+//isLastNumberEven          ---against--->odd final number
+const cardVailidator = (num) => {
+  const n = num.toString();
+  const isLengthSixteen = n.length === 16;
+  const isAllNumber = typeof Number(n) === 'number';
```

Comment:
This is not a valid condition to check if the card is all number, because `typeof Number(n)` will always return `'number'`. 

Please refer to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number. Note that `typeof NaN` is also equals to `'number'`